with-astro/scripts/validateAccessToken.ts

```diff
@@ -0,0 +1,253 @@
+import { Polar } from '@polar-sh/sdk'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+// Colors for console output
+const colors = {
+  green: '\x1b[32m',
+  red: '\x1b[31m',
+  yellow: '\x1b[33m',
+  blue: '\x1b[34m',
+  reset: '\x1b[0m',
+  bold: '\x1b[1m',
+}
+
+function log(message: string, color: string = colors.reset) {
+  console.log(`${color}${message}${colors.reset}`)
+}
+
+function logSuccess(message: string) {
+  log(`✅ ${message}`, colors.green)
+}
+
+function logError(message: string) {
+  log(`❌ ${message}`, colors.red)
+}
+
+function logWarning(message: string) {
+  log(`⚠️  ${message}`, colors.yellow)
+}
+
+function logInfo(message: string) {
+  log(`ℹ️  ${message}`, colors.blue)
+}
```

Comment:
i appreciate that but can we please keep it simple? i.e. console.warn, console.log